CMakePresets.json

```diff
@@ -12,6 +12,19 @@
         "CMAKE_C_FLAGS": "-g -O2 -Wall -fsanitize=address,undefined,nullability,float-divide-by-zero,integer-divide-by-zero,implicit-conversion -fno-sanitize-recover -fno-omit-frame-pointer -fsanitize-ignorelist=${sourceDir}/tools/developer/ignorelist.txt",
         "CMAKE_EXE_LINKER_FLAGS": "-fsanitize=address,undefined,nullability,float-divide-by-zero,integer-divide-by-zero,implicit-conversion -fsanitize-address-use-after-scope -fno-sanitize-recover"
       }
+    },
+    {
+      "name": "clang+coverage",
+      "displayName": "Clang coverage",
+      "description": "Configure with Clang for coverage",
+      "generator": "Unix Makefiles",
+      "binaryDir": "build-coverage",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "CMAKE_CXX_FLAGS": "-fprofile-instr-generate -fcoverage-mapping",
+        "CMAKE_C_FLAGS": "-fprofile-instr-generate -fcoverage-mapping",
```

Comment:
Do you not need to specify `-g -O0` in the FLAGS?

Reply:
Of course. Fixed.

Reply:
After playing with this a bit more, it looks like `-O1` is the lowest we can go and still have this run in anything close to a reasonable amount of time.